core-bundle/src/Resources/contao/dca/tl_form_field.php

```diff
@@ -151,6 +151,8 @@
 	(
 		'id' => array
 		(
+			'label'                   => array('ID'),
+			'search'                  => true,
```

Comment:
I don‘t think we need this for child tables, do we? Searching a form by ID makes sense, because you might stumble upon a form ID in an insert tag. But there is no use case for form field IDs in the core IIRC.

Reply:
It's likely used heavily for insert tags as well.

Reply:
There is no `insert_form_field` insert tag. These are the ones that use IDs:

<img width="416" alt="" src="https://github.com/contao/contao/assets/1192057/e6bcda0d-32b8-4c21-8da4-c2d8d25ec3ee">

Reply:
So articles, content elements, modules, forms, news and events should have `'search' => true`. The rest does not need it.

Reply:
In the core, yes. But very popular extensions provide insert tags for stuff like that. Imho, we should make it searchable by default. Even when communicating with customers I just tell them to check ID `<id>` of whatever element they should check 🤔

Reply:
Fine with me, too.